sympy/geometry/ellipse.py

```diff
@@ -537,6 +537,39 @@ def focus_distance(self):
 
         """
         return Point.distance(self.center, self.foci[0])
+   
+    def auxiliary_circle(self, x='x', y='y'):
+        """The equation of auxiliary circle of the ellipse.
+        
+        Parameters
+        ==========
+        
+        x : str, optional
+            Label for the x-axis. Default value is 'x'.
+        y : str, optional
+            Label for the y-axis. Default value is 'y'.
+               
+        Returns
+        =======
+        
+        equation : sympy expression
+        
+        References
+        ==========
+        
+        http://mathworld.wolfram.com/AuxiliaryCircle.html
+              
+        Examples
+        ========
+     
+        >>> from sympy import Ellipse, Point
+        >>> ellipse = Ellipse(Point(2, 4), 9, 1)
+        >>> ellipse.auxiliary_circle()
+        (x - 2)**2 + (y - 4)**2 - 9
```

Comment:
Can you change 9 to 81?

https://travis-ci.org/sympy/sympy/jobs/424933449#L2367-L2375

Reply:
thanks @sylee957